tests/Microsoft.Bot.Builder.AI.LUIS.Tests/TestData/DynamicListsAndList.json

```diff
@@ -44,7 +44,7 @@
         "DeltaAir"
       ]
     ],
-    "geographyV2": [
```

Comment:
>"geographyV2": [](start = 4, length = 14)

This is actually a change in the tests, i.e. this is a geography with a role--there is no longer geography without a role.